### PR TITLE
Makes a Named Mulebot gain a new name!

### PR DESCRIPTION
**Pull request by Bobbanz1**
## About The Pull Request
Replaces a Mulebots name with a new Name!

## Why It's Good For The Game
Named Mulebots that are named after people are not good

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl:
fix: Fixes a named Mulebot by giving it a new name!
/:cl:

### CODE REVIEW

**Review by Bokkiewokkie — DISMISSED** [review]

**Review by covertcorvid — CHANGES_REQUESTED** [review]
If we're giving it a dog name can we either go with the traditional "Fluffy" or something else like, I dunno, Fido or Rover or Spot or whatever?
It's been pointed out that "fluffer" has a porn-related meaning

**Comment by Bobbanz1** [comment]
> If we're giving it a dog name can we either go with the traditional "Fluffy" or something else like, I dunno, Fido or Rover or Spot or whatever? It's been pointed out that "fluffer" has a porn-related meaning

Done, it's now a Metal Penguin called Mittens

**Review by covertcorvid — DISMISSED** [review]

File: _maps/map_files/Tycoon/Tycoon2.dmm [hunk 1]
```diff
@@ -27092,8 +27092,8 @@
 	},
 /mob/living/simple_animal/bot/mulebot{
 	home_destination = "QM #2";
-	name = "Mobility Cube";
-	desc = "Some kind of weird cube. Sometimes it changes colors or shapes. We are not not sure what to think of it doing this, but it seems pretty harmless."
+	name = "Mittens";
+	desc = "For some reason this strange looking metal penguin seems to have a sticker attached to it with the words Mittens written on it!"
```
Comment (covertcorvid) [inline _maps/map_files/Tycoon/Tycoon2.dmm:27096]:
```suggestion
	desc = "For some reason this strange looking metal penguin seems to have a sticker attached to it with the word Mittens written on it!"
```
One more nitpick - Mittens is one word

Comment (Bobbanz1) [reply _maps/map_files/Tycoon/Tycoon2.dmm:27096]: 🤏

**Review by covertcorvid — APPROVED** [review]

**Pull request merged**